src/main/python/covata/delta/signer.py

```diff
@@ -115,7 +115,6 @@ def __get_signature_materials(method, url, headers, payload):
     cvt_date = datetime.utcnow().strftime(CVT_DATE_FORMAT)
     headers_ = dict(headers)
     headers_["Cvt-Date"] = cvt_date
-    headers_['Host'] = url_parsed.hostname
```

Comment:
I missed this - was our plan to remove this (I know it's not explicitly in the specifications).